labs/iot/gps-lab/server/code/server.pyproj

```diff
@@ -0,0 +1,80 @@
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
```

Comment:
Can we remove this and just focus on VS Code? Having this file may confuse VS Code users who won't know what it is for.